<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeAnnotation</class>
 <widget class="QDialog" name="ShapeAnnotation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imput annotation for region</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/24/annotate.png</normaloff>icons/24/annotate.png</iconset>
  </property>
  <property name="toolTip">
   <string>Input annotation information for shape</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Type of region</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Region Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Region ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Region Description</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editRegionDescription"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editRegionName"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editRegionID"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cboRegionType"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editRegionName</tabstop>
  <tabstop>editRegionID</tabstop>
  <tabstop>cboRegionType</tabstop>
  <tabstop>editRegionDescription</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShapeAnnotation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShapeAnnotation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
